### DISCUSSION TITLE
idea: request to load change log?

### DISCUSSION THREAD

**Discussion by shiiverstorm** [ImranR98/Obtainium#547] [Ideas]
I don't have a clue how complicated this might be but I love reading change logs lol. I know not every Dev includes them but it would be cool to have the most recent change log on the app page :) 

Thank you for your amazing work!! Obtanium is already gorgeous and super intuitive!

**Comment by ImranR98** [comment]
You can already see change logs (for supported sources) by clicking on the app's release date (on the main apps list page).
